sdk/core/Azure.Core.TestFramework/src/UseSyncMethodsInterceptor.cs

```diff
@@ -36,6 +36,12 @@ public UseSyncMethodsInterceptor(bool forceSync)
         [DebuggerStepThrough]
         public void Intercept(IInvocation invocation)
         {
+            if (invocation.Method.Name.Contains("WaitForCompletionAsync"))
```

Comment:
We don't currently instrument data-plane operations and ArmOperation has a method we can forward to. Why the exclude?